app/src/main/res/values/strings.xml

```diff
@@ -279,18 +279,18 @@
   <string name="nearby_location_has_not_changed">Location has not changed.</string>
   <string name="nearby_location_not_available">Location not available.</string>
   <string name="location_permission_rationale_nearby">Permission required to display a list of nearby places</string>
-  <string name="get_directions">GET DIRECTIONS</string>
+  <string name="get_directions">Get Directions</string>
   <string name="read_article">READ ARTICLE</string>
```

Comment:
This one also needs to be done, I am not able to check whole file now. Can you please cross over it one more time?

Reply:
i cross checked and all of them are done now